charts/open-notificaties/templates/deployment.yaml

```diff
@@ -85,3 +85,61 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "open-notificaties.workerFullname" . }}
+  labels:
+  {{- include "open-notificaties.workerLabels" . | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
this should be configurable to scale the number of workers depending on workload

Reply:
I thought about that. Are you sure that if there are 2 or more workers, messages of the queue are not delivered more than once? I will make this scalable.

Reply:
yes - the broker keeps track of them & the default deployment with Ansible fires up 3 replicas on both Docker-on-vm and Kubernetes ;)